packages/wallets/src/components/Base/WalletPasswordField/WalletPasswordField.tsx

```diff
@@ -73,7 +89,9 @@ const WalletPasswordField: React.FC<WalletPasswordFieldProps> = ({
             <WalletTextField
                 autoComplete={autoComplete}
                 errorMessage={isTouched && (passwordError ? passwordErrorMessage.PasswordError : errorMessage)}
-                isInvalid={(!validPassword(password) && isTouched) || passwordError}
+                isInvalid={
+                    (!(mt5Policy ? validPasswordMT5(password) : validPassword(password)) && isTouched) || passwordError
+                }
```

Comment:
hey @aizad-deriv 
could we extract it to a `(!(mt5Policy ? validPasswordMT5(password) : validPassword(password)) && isTouched) || passwordError` separate variable?